docs/validate-stable-rc-builds.md

```diff
@@ -0,0 +1,178 @@
+
+# Stable rc review validation process
+Following steps to be performed when a stable rc review pushed on stable mailing list
+
+## Wait for stable rc review email
+Greg KH pushes on to stable rc trees and creates a tag
+then sends out email on stable ML with the list of patches included in that
```

Comment:
Can you be more specific here about the process? Perhaps link "stable rc trees" to the actual repo. Same with mailing list.

Reply:
Added required links and description.